hera_sim/tests/test_cli_utils.py

```diff
@@ -37,6 +37,7 @@ def test_filing_params_parser():
         cli_utils.get_filing_params(bad_params)
 
 
+# TODO: break this up into multiple tests
```

Comment:
yes, here would be a good place to user parameterization. 

Should be like:
```python
@pytest.mark.parameterize(["freq", "time", "array"], itertools.prod(freq_params, time_params, array_params))
def test_config_validation(freq, time,array, tmp_path):
    ...
```

Reply:
I've refactored this testing module.